runtime/src/gpu/nvidia/Makefile.share

```diff
@@ -28,7 +28,7 @@ GPU_OBJS = $(addprefix $(GPU_OBJDIR)/,$(addsuffix .o,$(basename $(GPU_SRCS))))
 # a comma-separated list.
 RUNTIME_CXXFLAGS += -x cuda -Wno-unknown-cuda-version \
                     -Xclang -fcuda-allow-variadic-functions \
-                    --offload-arch=$(CHPL_MAKE_GPU_ARCH)
+                    --offload-arch=$(CHPL_MAKE_GPU_ARCH) -D__STRICT_ANSI__=1
```

Comment:
Was https://github.com/chapel-lang/chapel/pull/24303/files#diff-32227b333833b287cbe3c32f2ac41d8689fb69236c82f321fa039155c4cd5d09R72 not enough? It is not really a problem, but we might be adding this twice for `gpu-nvidia-reduce.cc`.

Reply:
yes you are right, this is extra with the changes in `compile_link_args_utils.py`. Removing from here.